.gitignore

```diff
@@ -64,6 +64,9 @@ typings/
 
 # End of https://www.gitignore.io/api/node
 
+# OSX
+.DS_Store
```

Comment:
Btw I personally I have this in my `~/.gitignore_global` to avoid making repos messy :P

Reply:
TIL!